base-theme/layouts/partials/course_list.html

```diff
@@ -3,6 +3,6 @@ <h1 class="course-list-title col-12 col-lg-8 pb-2">
   {{ $course_list.Title }}
 </h1>
 <div class="mb-3 col-12 col-lg-8 pb-2">
-  {{ $course_list.Description | $.RenderString }}
+  {{ $course_list.Content }}
```

Comment:
Do not have to use `.RenderString` anymore, as we are now rendering Content of markdown file.